escrow/SPECIFICATION.md

```diff
@@ -0,0 +1,95 @@
+Table of Content
+- [Actions that users are able to perform](#actions-that-users-are-able-to-perform)
+  - [Arbiter](#arbiter)
+    - [`resolve_dispute()`](#resolve_dispute)
+  - [Buyer](#buyer)
+    - [`accept_arbiter()`](#accept_arbiter)
+    - [`deposit()`](#deposit)
+    - [`dispute()`](#dispute)
+    - [`transfer_to_seller()`](#transfer_to_seller)
+  - [Seller](#seller)
+    - [`create_escrow()`](#create_escrow)
+    - [`propose_arbiter()`](#propose_arbiter)
+    - [`return_deposit()`](#return_deposit)
+    - [`take_payment()`](#take_payment)
+
+# Overview
+
+This document provides an overview of the application.
+
+It outline the use cases, i.e. desirable functionality, in addition to requirements for the smart contract and the user interface.
+
+# Use Cases
+
+This section contains general information about the functionality of the application and thus does not touch upon any technical aspects.
+
+If you are interested in a functional overview then this is the section for you.
+
+## Actions that users are able to perform
+
+This sub-section details what a user is able to do e.g. click a button and "x, y, z" happens.
+
+### Arbiter
+
+#### `resolve_dispute()`
+
+1. When an escrow is in dispute the arbiter of the escrow can perform a resolution in either direction of the buyer or seller
```

Comment:
```suggestion
1. When an escrow is in dispute the arbiter of the escrow can perform a resolution in either the direction of the buyer or seller
```

Reply:
This is incorrect

Reply:
I plugged both options into an online grammar tool and they both work.  I guess it's just a matter of opinion.